polling/periodic_sequence/README.md

```diff
@@ -0,0 +1,17 @@
+# Periodic Child Workflow Retry
```

Comment:
Inconsistent title casing at top of README (but not that important)

Reply:
Unresolving. This H1 title casing (capitalize each word) is different than other README H1 casing (capitalize only first word).

Reply:
Fixed